protocol/messenger.go

```diff
@@ -2000,37 +1982,30 @@ func (m *Messenger) ConfirmMessagesProcessed(messageIDs [][]byte) error {
 	return nil
 }
 
-// DEPRECATED: required by status-react.
 func (m *Messenger) MessageByID(id string) (*Message, error) {
```

Comment:
Why are they not deprecated anymore?

Reply:
I think the idea is was that we will build a new API from scratch with non-deprecated methods, after a discussion we agreed on not marking endpoint as deprecated until we have a suitable replacement or there are no more use cases for it. Currently this endpoints are used by status-react. (also before we had two tables for messages, now there's only one)

Reply:
Makes sense.